tests/behat.yml.dist

```diff
@@ -50,12 +50,12 @@ default:
         Modal content: '#drupal-modal'
         Modal buttons: '.ui-dialog-buttonpane'
         Navigation menu block: '#block-navigation'
+        Navigation bottom menu block: '#block-navigation-bottom'
```

Comment:
Let's call this the "Table of contents" instead of the "Navigation bottom menu block".